apps/forms-flow-ai/forms-flow-web/package.json

```diff
@@ -77,7 +77,10 @@
     "source-map-explorer": "^2.5.2",
     "stompjs": "^2.3.3",
     "typeface-nunito-sans": "0.0.72",
-    "ws": "^7.4.6"
+    "ws": "^7.4.6",
```

Comment:
Hopefully this works, even with our old package-lock.json (pre-canvas). 

I checked out your branch and docker built formsflowweb.  It failed with the current package-lock.json (as expected), and when I copied in the [older "working"](https://raw.githubusercontent.com/bcgov/foi-flow/a72ae56f95a867a8f1be18e51bd2bcc7f60168ec/apps/forms-flow-ai/forms-flow-web/package-lock.json) one, and it failed.  Uh oh, as we feared, we can't include any more dependencies like this until we figure out the package-lock.json issue.

```
 > [build-stage 10/10] RUN npm run build:                                                                                                                    
#17 0.788                                                                                                                                                    
#17 0.788 > formsflow-ai-react@1.0.0 build /forms-flow-web/app                                                                                               
#17 0.788 > react-scripts --max_old_space_size=6144 build
#17 0.788 
#17 4.510 Creating an optimized production build...
#17 158.8 The build failed because the process exited too early. This probably means the system ran out of memory or someone called `kill -9` on the process.
#17 159.2 npm ERR! code ELIFECYCLE
#17 159.2 npm ERR! errno 1
#17 159.3 npm ERR! formsflow-ai-react@1.0.0 build: `react-scripts --max_old_space_size=6144 build`
#17 159.3 npm ERR! Exit status 1
#17 159.3 npm ERR! 
#17 159.3 npm ERR! Failed at the formsflow-ai-react@1.0.0 build script.
#17 159.3 npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
#17 159.3 
#17 159.3 npm ERR! A complete log of this run can be found in:
#17 159.3 npm ERR!     /root/.npm/_logs/2021-07-28T16_39_47_467Z-debug.log
```